projects/angular/src/timeline/_timeline.clarity.scss

```
@@ -190,30 +260,42 @@
       word-break: break-word;
       width: timeline-variables.$clr-timeline-vertical-step-header-min-width;
       min-width: timeline-variables.$clr-timeline-vertical-step-header-min-width;
-      margin-right: timeline-variables.$clr-timeline-vertical-step-header-right-padding;
-      margin-top: timeline-variables.$clr-timeline-vertical-step-header-top-padding;
       margin-bottom: 0;
+      @include mixins.css-var(
+        margin-right,
+        clr-timeline-step-internal-margin,
+        timeline-variables.$clr-timeline-vertical-step-header-right-padding,
+        variables.$clr-use-custom-properties
+      );
+      @include mixins.css-var(
+        margin-top,
+        clr-timeline-step-internal-margin,
+        timeline-variables.$clr-timeline-vertical-step-header-top-padding,
+        variables.$clr-use-custom-properties
+      );
     }
 
     .clr-timeline-step-title {
       margin-top: 0;
+      @include mixins.css-var(margin-top, clr-timeline-step-internal-margin, 0, variables.$clr-use-custom-properties);
     }
 
     .clr-timeline-step-body {
       display: flex;
       flex-direction: column;
       min-width: timeline-variables.$clr-timeline-vertical-step-min-width;
-      margin-left: timeline-variables.$clr-timeline-vertical-step-body-left-padding;
-      margin-top: timeline-variables.$clr-timeline-vertical-step-body-top-padding;
-    }
-  }
-
-  @keyframes spin {
-    0% {
-      transform: rotate(0deg);
-    }
-    100% {
-      transform: rotate(360deg);
+      @include mixins.css-var(
+        margin-left,
+        clr-timeline-step-internal-margin,
+        timeline-variables.$clr-timeline-vertical-step-body-left-padding,
+        variables.$clr-use-custom-properties
+      );
+      @include mixins.css-var(
+        margin-top,
+        clr-timeline-internal-margin-0,
+        timeline-variables.$clr-timeline-vertical-step-body-top-padding,
+        variables.$clr-use-custom-properties
+      );
```

I would expose general margin token and use it like that, this `*-margin-0` doesn't look right.
```suggestion
      @include mixins.css-var(
        margin,
        clr-timeline-step-internal-margin,
        timeline-variables.$clr-timeline-vertical-step-body-top-padding 0 0 timeline-variables.$clr-timeline-vertical-step-body-left-padding,
        variables.$clr-use-custom-properties
      );
```

---

corrected in commit 0a5d649